docassemble/EFSPIntegration/efm_client.py

```diff
@@ -68,14 +68,17 @@ def __init__(self, url:str=None, api_key:str=None):
     self.verbose = False
     self.authed_user_id = None
     self.set_verbose_logging(True)
-
+    self.credentials_code_block = credentials_code_block
+    
   def _call_proxy(self, send_func):
     try:
       resp = send_func()
       if resp.status_code == 401:
-        auth_resp = self.authenticate_user()
-        if auth_resp.response_code == 200:
-          resp = send_func()
+        # auth_resp = self.authenticate_user()
+        # if auth_resp.response_code == 200:
+        #  resp = send_func()
+        #else:
+        reconsider(self.credentials_code_block)
```

Comment:
I guess it doesn't need to be:
```suggestion
        reconsider(self.credentials_code_block)        
        if auth_resp.response_code == 200:
          resp = send_func()
```
because the reconsider block we come back and execute the call automatically? Kinda weird and hard to intuit from the code, but I guess that's fine.